Demo/EarlGreyExample/EarlGreyExampleSwift/ViewController.swift

```diff
@@ -46,38 +46,39 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
     let views = ["clickMe": clickMe, "send": send, "send2": send2, "table": table,
         "sendMessageView": sendMessageView]
     var allConstraints = [NSLayoutConstraint]()
-    let verticalConstraints = NSLayoutConstraint.constraintsWithVisualFormat(
-        "V:|-40-[clickMe]-40-[send2]-40-[table]|", options: [], metrics: nil, views: views)
+    let verticalConstraints = NSLayoutConstraint.constraints(
+        withVisualFormat: "V:|-40-[clickMe]-40-[send2]-40-[table]|",
```

Comment:
Is that necessary? Since we only use them in these tests. Would you like me to move the whole format instead?